package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "start-server-webpack-plugin",
-  "version": "3.0.0-rc3",
+  "name": "@fivethreeo/start-server-webpack-plugin",
```

Comment:
Better not ;)

Reply:
About version, github packages only allows 3.0.0-alpha.x 3.0.0-beta.x. 3.0.0-rcx was not allowed I am no semver expert ;)